### ISSUE TITLE
Accessibility: Combobox:When initial letter is typed, pop up with corresponding result list appears, but information about result list pop up such as “has pop up” or “2 result list items” is not conveyed. 

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3451]
Description: When initial letter is typed, pop up with corresponding result list appears, but information about result list pop up such as “has pop up” or “2 result list items” is not conveyed. 

Expected: Information about result list items/pop up should be conveyed to the user

Screenshot:

![image](https://user-images.githubusercontent.com/32538291/94521938-72b27480-024c-11eb-9d39-718372d4dec1.png)

**Comment by kavya-b** [comment]
Hi @puru-hk can you verify if this issue still persists? Some a11y issues were addressed in PR #4353 in Jan '21, and in macos voiceover, screen readers announce 'has popup' and on navigating the list, also announces '1 of 8'.

**Comment by kavya-b** [comment]
@puru-hk @Seamoo13 could you please verify if this issue still exists?

**Comment by puru-hk** [comment]
@Sanjay-1312 I have tested it is working fine. Could you verify once.

**Comment by Sanjay-1312** [comment]
Hello Colleagues,

On entering data the pop will appear and filter results will appear the information such as pop up and the result item count is not read.
![image](https://user-images.githubusercontent.com/78714518/122043625-2b2f4100-cdf9-11eb-908c-14d5ec4d7b61.png)
EB: The searched Filtered resulted has pop up and the it has item count result should be read.

URL: https://fundamental-ngx.netlify.app/#/platform/combobox
JAWS 2021

**Comment by Betrozov** [comment]
Hi @puru-hk @Seamoo13
could you please verify if this issue still exists?
I can not reproduce it unfortunately

**Comment by Seamoo13** [comment]
@Betrozov I don't have access to JAWS, @puru-hk please check this one.

**Comment by puru-hk** [comment]
@Betrozov Yes able to reproduce as mentioned above

**Comment by puru-hk** [comment]
@Betrozov Yes able to reproduce as mentioned above

**Comment by Sanjay-1312** [comment]
Hello Colleagues,

The combo box can be closed using Esc key.

Retest details:
https://deploy-preview-6941--fundamental-ngx.netlify.app
JAWS 2021
